From a5147e457defc2fd036bdffd0967e0ec42638ec1 Mon Sep 17 00:00:00 2001
From: pfhchaos <pfh.chaos@gmail.com>
Date: Sat, 31 Dec 2016 15:31:24 -0800
Subject: [PATCH] fix to compile

---
 src/Main.hh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Main.hh b/src/Main.hh
index 69cf9cf..badcdd9 100644
--- a/src/Main.hh
+++ b/src/Main.hh
@@ -57,7 +57,7 @@ public:
             iteration_order.emplace_back(app.first, app.second);
         }
 
-        std::locale locale(suffixes.locale);
+        std::locale locale(suffixes.locale.c_str());
         std::sort(iteration_order.begin(), iteration_order.end(), [locale](
             const std::pair<std::string, const Application *> &s1,
             const std::pair<std::string, const Application *> &s2) {
-- 
2.11.0

