diff -r mednafen.orig/src/compress/Makefile.am.inc mednafen/src/compress/Makefile.am.inc
--- mednafen.orig/src/compress/Makefile.am.inc	2014-12-03 19:07:30.000000000 -0500
+++ mednafen/src/compress/Makefile.am.inc	2015-02-04 07:58:43.820369198 -0500
@@ -1 +1 @@
-mednafen_SOURCES	+=	compress/minilzo.c compress/ioapi.c compress/unzip.c compress/ZLInflateFilter.cpp
+mednafen_SOURCES	+=	compress/minilzo.c compress/ioapi.c compress/ZLInflateFilter.cpp
diff -r mednafen.orig/src/file.cpp mednafen/src/mednafen/file.cpp
--- mednafen.orig/src/file.cpp	2014-12-11 19:41:12.000000000 -0500
+++ mednafen/src/file.cpp	2015-02-04 07:58:51.124369477 -0500
@@ -29,7 +29,7 @@
 #include <errno.h>
 #include <trio/trio.h>
 
-#include "compress/unzip.h"
+#include <minizip/unzip.h>
 
 #include "file.h"
 #include "general.h"
diff -ur mednafen.orig/src/Makefile.am mednafen/src/Makefile.am
--- mednafen.orig/src/Makefile.am	2014-12-03 19:08:21.000000000 -0500
+++ mednafen/src/Makefile.am	2015-02-04 07:58:38.296368988 -0500
@@ -6,7 +6,7 @@
 bin_PROGRAMS	=	mednafen
 
 mednafen_SOURCES 	= 	debug.cpp error.cpp mempatcher.cpp settings.cpp endian.cpp mednafen.cpp git.cpp file.cpp general.cpp memory.cpp netplay.cpp state.cpp state_rewind.cpp movie.cpp player.cpp PSFLoader.cpp tests.cpp qtrecord.cpp Stream.cpp MemoryStream.cpp FileStream.cpp GZFileStream.cpp IPSPatcher.cpp
-mednafen_LDADD 		= 	trio/libtrio.a
+mednafen_LDADD 		= 	trio/libtrio.a -lminizip
 mednafen_DEPENDENCIES	=	trio/libtrio.a
 
 if HAVE_SDL
