--- mednafen.orig/src/psx/psx.cpp	2015-02-04 06:25:18.288155574 -0500
+++ mednafen/src/psx/psx.cpp	2015-02-04 06:05:39.880110665 -0500
@@ -1086,7 +1086,11 @@
  SPU->StartFrame(espec->SoundRate, MDFN_GetSettingUI("psx.spu.resamp_quality"));
 
  Running = -1;
+#if PSX_DBGPRINT_ENABLE
  timestamp = CPU->Run(timestamp, psf_loader == NULL && psx_dbg_level >= PSX_DBG_BIOS_PRINT, psf_loader != NULL);
+#else
+ timestamp = CPU->Run(timestamp, psf_loader == NULL, psf_loader != NULL);
+#endif
 
  assert(timestamp);
 
